File: Programar em C#/15-Switch.cs
// Programar em Python #15 - Switch

using System;

namespace Base {
    class Switch {
        public void Run() {
            char caso = 'D';

            switch (caso) {
                case 'A':
                    Console.WriteLine("O caso A existe!");
                    break;
                case 'B':
                    Console.WriteLine("O caso B existe!");
                    break;
                case 'C':
                    Console.WriteLine("O caso C existe!");
                    break;
                default:
                    Console.WriteLine($"O caso {caso} não existe!");
                    break;
            }
        }
    }
}

File: Programar em Java/15-Switch.java
// Programar em Java #15 - Switch

package com.caffeinealgorithm.programaremjava;

public class Switch {
    public void Run() {
        char caso = 'D';

        switch (caso) {
            case 'A':
                System.out.println("O caso A existe!");
                break;
            case 'B':
                System.out.println("O caso B existe!");
                break;
            case 'C':
                System.out.println("O caso C existe!");
                break;
            default:
                System.out.println(new StringBuilder().append("O caso ").append(caso).append(" não existe!"));
                break;
        }
    }
}